logstash-core-event-java/src/main/java/com/logstash/cbor/BinaryStruct.java

```diff
@@ -0,0 +1,28 @@
+package com.logstash.cbor;
```

Comment:
Sorry, my bad. I thought I had deleted that class. I was experimenting with a struct like object but then opted for a HashMap to carry the data and metadata.

Will remove.